pallets/author-filter/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+authors = ["PureStake"]
+edition = "2018"
+name = "pallet-author-filter"
+version = "0.1.0"
+
+[dependencies]
+parity-scale-codec = { version = "1.3.0", default-features = false, features = ["derive"] }
+serde = { version = "1.0.101", optional = true, features = ["derive"] }
```

Comment:
where is serde used? is there a genesis config?

Reply:
Seems to compile fine without serde. Good catch. I copied that from some other pallet as a starting point.

Is serde only necessary in a pallet when it has a genesis config? I didn't know that. Makes sense though so you can express the config in json.

Reply:
> Is serde only necessary in a pallet when it has a genesis config? I didn't know that. Makes sense though so you can express the config in json.

Yes, that is my understanding